### DISCUSSION TITLE
Backup

### DISCUSSION THREAD

**Discussion by jeng37** [Blackymas/NSPanel_HA_Blueprint#2135] [General]
Hello, 
I've been using this stuff for a year now.
Everything perfect!
But is there a way, if you need to update the blueprint, to somehow make a backup of the settings?
I had to reconfigure everything every time after the blueprint update.

**Comment by edwardtfn** [comment]
The update process shouldn't affect your existing settings... Are you doing anything different than the re-import as explained in our docs?

https://github.com/Blackymas/NSPanel_HA_Blueprint/blob/main/docs/howto.md#update-blueprint

**Reply by jeng37** [reply]
Yes, unfortunately the reimport function didn't work the last two times (version changes), so unfortunately I don't know the error message anymore.
I always had to rename the blueprint to xxx_old. And then I was able to install the new one.
But okay, I'll test it next time and document it carefully if it happens again.
Many thanks for the answer.

**Reply by edwardtfn** [reply]
On your automations.yaml file, you shoukd be able to find the old automation with all their settings... Copying the entire there and adding to the new automation (also in the same file) and then reloading automations shoukd do the work.

And please share the details the next time you have an issue re-importing. I'd like to work at that to make sure everything works smoothly.

**Comment by jeng37** [comment]
It worked this way.
Excellent!
Many Thanks ;-)
